src/app/view/components/_common/dot-icon-button/dot-icon-button.component.scss

```diff
@@ -114,6 +112,24 @@
     }
 
     button {
-        @include icon-button;
+        align-items: center;
+        background: transparent;
+        border-radius: 50%;
+        border: none;
+        color: $gray;
+        cursor: pointer;
+        display: inline-flex;
+        justify-content: center;
+        padding: 0.608rem; // 39x39 the defauld field height
```

Comment:
global variable?

Reply:
We can't because is a very specific number to match 39x39